test/tutorial/step-6-asynchronous-testing.md

```diff
@@ -0,0 +1,49 @@
+### Asynchronous Testing
+
+  Now let's restart the server `npm run start` and run those tests again, `npm run test`
+
+  Oh dear, another problem
+
+  ![async_failures](../assets/async_failures.png)
+
+
+Any guesses on what might be happening?
+
+_____
+
+### What is the issue??
+The final line "asynchronous operations weren't stopped" might give us a clue.
+
+Remember how Node is asynchronous? Well the same goes for our tests, they also need to handle requests asynchronously.
+
+Luckily it's pretty easy with Jest, we need to use the `async`, `await` and `done` keywords.
+
+Let's re-write our test to include these keywords
+
+___
+
+```
+it('should return 200 status on the GET endpoint', async done => {
+    const response = await request.get('/get-posts')
+
+    expect(response.status).toBe(200)
+
+    done()
+})
+```
+
+Let's break this down:
+ - The `async` function is always coupled with an `await` expression.
+ - The `async` function will wait until the `await` expression has completed, in our case the get request. Once that has completed the function will continue. 
```

Comment:
I'm wondering if it would be helpful to use the callback stuff in the main tutorial as a point of comparison, as both are ways are dealing with asynchronous code.  
No strong opinions though, it might just be overkill to go down that route 🤷

Reply:
I had a little fiddle around with this and I added a line that mentions callbacks used in the workshop. I wasn't sure how in depth to go but I figure if it's mentioned, students can go back to the Express Workshop to remind themselves about callbacks and asynchronous code. Let me know what you think.